docs/UserGuide.md

```diff
@@ -357,8 +371,8 @@ Format: `find KEYWORDS`
 * Searches return meetings matching at least one keyword, e.g. `Complete game` will return `Complete features` and `Final game`.
 
 Examples:
-* `find discuss user guide John` returns a meeting with title `discuss user guide` and another with description `with Mr John`.
-* `find Mcdonalds` returns a meeting with title `Meet Mcdonalds` and another with description `Cater Mcdonalds breakfast`.
+* `find Survey` returns a meeting with title `Survey potential customers` and another with description `Don't forget to present survey results.`.
```

Comment:
Here and every example in the UG: Is it safe to assume that there exist seed data in the app that corresponds to the UG so users can immediately work with them?